modules/channel-web/assets/examples/embedded-webchat.html

```diff
@@ -150,6 +138,65 @@ <h2>Control User Identity</h2>
       document.getElementById('toggle-bp').addEventListener('click', function() {
         window.botpressWebChat.sendEvent({ type: webchatOpen ? 'hide' : 'show' })
       })
+
+      let secondInit = false
+      const secondChatId = 'second'
+      function toggleChat2() {
+        window.setTimeout(function() {
+          window.window.botpressWebChat.sendEvent({ type: 'toggle' }, secondChatId)
+        }, 100)
+      }
+      document.getElementById('show-second-bp').addEventListener('click', function() {
+        if (!secondInit) {
+          const config = {
+            host: window.location.origin || 'http://localhost:3000',
+            botId: botId,
+            userId: Math.random()
+              .toString()
+              .slice(-4),
+            showConversationsButton: false,
+            botName: 'Chat window #2',
+            className: 'chat2',
+            enableReset: false,
+            chatId: secondChatId,
+            hideWidget: true,
+            disableAnimations: true,
+            containerWidth: 700,
+            layoutWidth: 700,
+            exposeStore: true
+          }
+          window.botpressWebChat.init(config, '#chat2')
+          secondInit = true
+        } else {
+          toggleChat2()
+        }
```

Comment:
Can you invert that big if and return early instead ?

Reply:
done